### PR TITLE
Prepare tests for Fedora 32

### PR DESCRIPTION
**Pull request by tyll**
Fix some issues found when working on #219

### CODE REVIEW

File: tests/playbooks/tests_checkpoint_cleanup.yml [hunk 1]
```diff
@@ -18,6 +18,10 @@
     - linux-system-roles.network
   tasks:
     - block:
+        # Workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1832897
+        - package:
+            name: dbus-tools
+            state: present
```
Comment (richm) [inline tests/playbooks/tests_checkpoint_cleanup.yml:24]: Is this needed if not using initscripts?  I can't tell if this playbook includes `nm` specific tasks, or perhaps it is needed in either case?
Is this package needed and provided on EL6 through EL8?  Or is this playbook only executed on certain platforms?

Comment (tyll) [reply tests/playbooks/tests_checkpoint_cleanup.yml:24]: This playbook does not run on EL6. The test uses both initscripts and nm to trigger an error condition but this only works when dbus-tools is present. initscripts does not really need dbus-tools it is only needed when NM is also running to tell NM not to touch an interface.

File: tests/playbooks/tests_checkpoint_cleanup.yml [hunk 2]
```diff
@@ -67,6 +73,9 @@
                   - name: "{{ interface }}"
                     persistent_state: absent
                     state: down
+            - file:
+                dest: /etc/sysconfig/network-scripts/ifcfg-cptstbr
+                state: absent
```
Comment (pcahyna) [inline tests/playbooks/tests_checkpoint_cleanup.yml:78]: Is this also a workaround for some bug? otherwise lgtm.

Comment (tyll) [reply tests/playbooks/tests_checkpoint_cleanup.yml:78]: It is the quick cleanup for the initscripts provider. But now that I look at it again, I noticed it is missing the `{{ interface }}` expansion.

**Pull request merged**